README.md

```diff
@@ -358,3 +360,56 @@ Please check out common DSC Community [contributing guidelines](https://dsccommu
 
 - [CMAssetIntelligencePoint_Absent](Source\Examples\Resources\CMAssetIntelligencePoint\CMAssetIntelligencePoint_Absent.ps1)
 - [CMAssetIntelligencePoint_Present](Source\Examples\Resources\CMAssetIntelligencePoint\CMAssetIntelligencePoint_Present.ps1)
+
+### CMDistributionPoint
+
+- **[String] SiteCode** _(Key)_: Specifies the Site Code for the Configuration
+  Manager site.
+- **[String] SiteServerName** _(Key)_: Specifies the SiteServer to install the role.
+- **[String] Description** _(Write)_: Specifies a description for the
+  distribution point.
+- **[UInt32] MinimumFreeSpaceMB** _(Write)_: Specifies the amount of free space
+  to reserve on each drive used by this distribution point.
+  Only used when distribution point is not currently installed.
+- **[String] PrimaryContentLibraryLocation** _(Write)_: Specifies the primary
+  content location. Configuration Manager copies content to the primary content location
+  until the amount of free space reaches the value that you specified.
+  Only used when distribution point is not currently installed.
+- **[String] SecondaryContentLibraryLocation** _(Write)_: Specifies the
+  secondary content location.
+  Only used when distribution point is not currently installed.
+- **[String] PrimaryPackageShareLocation** _(Write)_: Specifies the primary
+  package share location. Configuration Manager copies content to the primary package
+  share location until the amount of free space reaches the value that you specified.
+  Only used when distribution point is not currently installed.
+- **[String] SecondaryPackageShareLocation** _(Write)_: Specifies the secondary
+  package share location.
+  Only used when distribution point is not currently installed.
+- **[DateTime] CertificateExpirationTimeUtc** _(Write)_: Specifies, in UTC format,
+  the date and time when the certificate expires.
+  Only used when distribution point is not currently installed.
+- **[String] ClientCommunicationType** _(Write)_: Specifies protocol clients or devices
+  communicate with the distribution point.
+  - Values include: { Http | Https }
+- **[String] BoundaryGroups[]** _(Write)_: Specifies an array of boundary groups
+  by name.
+- **[String] BoundaryGroupStatus** _(Write)_: Specifies if the boundary group is
```

Comment:
I would add something here indicating that this won't actually create boundary groups, rather add/remove/match boundary groups on the DP.